deploys/demo-test.syncfor.science/nginx-ssl-proxy/docker-compose.yml

```diff
@@ -32,7 +32,7 @@ services:
       - certs:/etc/nginx/certs:ro
     entrypoint: /usr/local/bin/docker-gen -notify-sighup nginx_proxy -watch -only-exposed -wait 5s:30s /etc/docker-gen/templates/nginx.tmpl /etc/nginx/conf.d/default.conf
   lets-encrypt:
-    image: jrcs/letsencrypt-nginx-proxy-companion
+    image: alastaircoote/docker-letsencrypt-nginx-proxy-companion
```

Comment:
I had to blow away everything, and then I ran into a bug in the letsencrypt image. It looks like jrcs stopped maintaining it, and alastairecoote forked it and took it over. https://github.com/JrCs/docker-letsencrypt-nginx-proxy-companion/issues/130#issuecomment-266772072